packages/website/package.json

```diff
@@ -16,7 +16,7 @@
         "deploy_staging":
             "npm run build; aws s3 sync ./public/. s3://staging-0xproject --profile 0xproject --region us-east-1 --grants read=uri=http://acs.amazonaws.com/groups/global/AllUsers",
         "deploy_live":
-            "npm run build; aws s3 sync ./public/. s3://0xproject.com --profile 0xproject --region us-east-1 --grants read=uri=http://acs.amazonaws.com/groups/global/AllUsers --exclude *.map.js"
+            "DEPLOY_ROLLBAR_SOURCEMAPS=true npm run build; aws s3 sync ./public/. s3://0xproject.com --profile 0xproject --region us-east-1 --grants read=uri=http://acs.amazonaws.com/groups/global/AllUsers --exclude *.map.js"
```

Comment:
should we do this for dogfood as well, and add more types to the `Environments` enum for staging and dogfood?

Reply:
+1 for environments, but dogfood doesn't need the rollbar upload since we upload the sourcemaps with the bundle. Notice `--exclude *.map.js` at the end of `deploy_live`.